src/components/Images.js

```diff
@@ -0,0 +1,118 @@
+/* eslint react/display-name: 0 */
+import React from "react";
+
+const Images = {
```

Comment:
why not have svgs as svg files and import them properly?

https://github.com/twopluszero/next-images